assets/terraform-modules/matchbox-flatcar/profiles.tf

```
@@ -53,44 +48,32 @@ resource "matchbox_profile" "cached-flatcar-linux-install" {
 
   args = flatten([
     "initrd=flatcar_production_pxe_image.cpio.gz",
-    "ignition.config.url=${var.matchbox_http_endpoint}/ignition?uuid=$${uuid}&mac=$${mac:hexhyp}",
+    "ignition.config.url=${var.http_endpoint}/ignition?uuid=$${uuid}&mac=$${mac:hexhyp}",
     "flatcar.first_boot=yes",
     "console=tty0",
     "console=ttyS0",
     var.kernel_args,
   ])
 
-  container_linux_config = templatefile("${path.module}/cl/install.yaml.tmpl", {
+  container_linux_config = templatefile("${path.module}/templates/install.yaml.tmpl", {
     os_channel               = var.os_channel
     os_version               = var.os_version
-    ignition_endpoint        = format("%s/ignition", var.matchbox_http_endpoint)
+    ignition_endpoint        = format("%s/ignition", var.http_endpoint)
     install_disk             = var.install_disk
     container_linux_oem      = var.container_linux_oem
     ssh_keys                 = jsonencode(var.ssh_keys)
     install_to_smallest_disk = var.install_to_smallest_disk
     # profile uses -b baseurl to install from matchbox cache
-    baseurl_flag = "-b ${var.matchbox_http_endpoint}/assets/flatcar"
+    baseurl_flag = "-b ${var.http_endpoint}/assets/flatcar"
   })
 }
 
-// Kubernetes Controller profiles
-resource "matchbox_profile" "controllers" {
-  count = length(var.controller_names)
+// Node profiles
```

Or maybe we can remove the comment completely, it does not bring any value IMO.
```suggestion
// Node profiles.
```